packages/snap/src/utils/types.ts

```diff
@@ -1,17 +1,32 @@
 import * as t from 'io-ts'
 
-export const SimpleTransaction = t.intersection([
-    t.type({
-        to: t.string,
-        value: t.string
-    }),
-    t.partial({
-        feeCurrency: t.union([t.string, t.undefined])
-    })
+export const TransactionConfig = t.partial({
+    from: t.union([t.string, t.number]),
+    to: t.string,
+    value: t.union([t.number, t.string, t.bigint]),
```

Comment:
are we sure about bigInt here?

Reply:
Please see comment https://github.com/celo-org/gas-snap/pull/17#discussion_r1238822859